content/docs/ai/ai-intro.md

```diff
@@ -7,18 +7,128 @@ updatedOn: '2023-10-07T10:43:33.361Z'
 
 Vector databases enable efficient storage and retrieval of vector data, which is an essential component in building AI applications that leverage Large Language Models (LLMs) such as OpenAI.
 
-Neon supports the `pgvector` open-source extension, which enables Postgres as a vector database for storing and querying vector embeddings. By enabling Postgres as a vector database, you can keep your data in the open source database that you know and trust. There's no need for data migration or a proprietary vector storage solution.
+Neon supports the `pgvector` open-source extension, which enables Postgres as a vector database for storing and querying vector embeddings. By enabling Postgres as a vector database, you can keep your data in the open-source database that you know and trust. There's no need for data migration or a proprietary vector storage solution.
```

Comment:
proprietary -> 3rd party?

Reply:
That's probably more relatable language. Changed!